### PR TITLE
Add support for unassociated streams

### PR DESCRIPTION
**Pull request by idastambuk**
**What this PR does / why we need it**:
When querying by propertyAlias was first implemented, it had support for unassociated streams ("disassociated" in AWS). 
Unassociated streams are data streams that are not associated with a propertyId on an asset, but contain historical data about a property all the same. They can be queried by a propertyAlias. 
You can see them here in the console if you apply the Disassociated Streams filter: 

Screenshot 2023-10-13 at 7 13 43 PM

[The 500 error from the bug report](https://github.com/grafana/iot-sitewise-datasource/issues/228) happened because of a null pointer reference since, when we query DescribeAssetPropertyWithContext for unassociated propertyAliases, it doesn't return an assetId or propertyId. The bug was introduced when we implemented fetching assets and property values only by propertyAlias. However, this only worked for associated streams, where DescribeAssetPropertyWithContext will actually return propertyId.

```
func getAssetIdAndPropertyId(query models.AssetPropertyValueQuery, client client.SitewiseClient, ctx context.Context) (models.AssetPropertyValueQuery, error) {
	result := query
	if query.PropertyAlias != "" {
		resp, err := client.DescribeTimeSeriesWithContext(ctx, &iotsitewise.DescribeTimeSeriesInput{
			Alias: getPropertyAlias(query.BaseQuery),
		})
		if err != nil {
			return models.AssetPropertyValueQuery{}, err
		}
		---> result.AssetIds = []string{*resp.AssetId}
		result.PropertyId = *resp.PropertyId
		
```
Screenshot 2023-10-16 at 12 13 14 PM

On the frontend, I fixed the panel header to show propertyAlias in the panel header when collapsed
Screenshot 2023-10-16 at 12 11 40 PM

I also moved some functions in the util package from /api, since I needed them in other packages

**Which issue(s) this PR fixes**:

### How to test: 
Add the following propertyAliased into the property alias field and test different "Get value ..." query types. For data, select "last year" in order to be able to see data, then you can zoom in. 

We have a few aliased streams in our dev account: 
turbine/1/torque -> disassociated - doesn't have an assetId/propertyId tied to it, so this can be tested for a fix
/turbine/1/wind_direction -> associated - for comparison

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by fridgepoet** [comment]
Heya @idastambuk just trying to understand what is expected here. I've tried "Get property value history" on some dissociated data streams in our squad dev account, but I'm getting a panic in the backend code still. This seems to be the same as what's currently on v1.11.1 
I think it's panicking here https://github.com/grafana/iot-sitewise-datasource/pull/231/files#diff-670b06752c76d1a93e8f926fe7568b7de2bce83536b701c5de5d723a94e4676eR27
Is that expected?
Let me know if I've got something wrong or missed something!

ex:
Screenshot 2023-10-17 at 18 08 53

**Review by kevinwcyu — APPROVED** [review]

**Comment by idastambuk** [comment]
> Heya @idastambuk just trying to understand what is expected here. I've tried "Get property value history" on some dissociated data streams in our squad dev account, but I'm getting a panic in the backend code still. This seems to be the same as what's currently on v1.11.1 I think it's panicking here as `h` is empty https://github.com/grafana/iot-sitewise-datasource/pull/231/files#diff-670b06752c76d1a93e8f926fe7568b7de2bce83536b701c5de5d723a94e4676eR53 Is that expected? Let me know if I've got something wrong or missed something!
> 
> ex: Screenshot 2023-10-17 at 18 08 53

Hey @fridgepoet, good catch, it only happens with property value history queries and when there is no data. I added a condition and updated the tests.

File: pkg/framer/property_value.go [hunk 1]
```diff
@@ -19,9 +19,9 @@ func (p AssetPropertyValue) Frames(ctx context.Context, resources resource.Resou
 	if err != nil {
 		return nil, err
 	}
-
+	var property *iotsitewise.DescribeAssetPropertyOutput
```
[the remark this answers is not in the document]
Comment (idastambuk) [reply pkg/framer/property_value.go]: Who knows what it did, but fixed it now!🙏🏻

File: pkg/server/test/property_value_aggregate_test.go [hunk 2]
```diff
@@ -195,4 +196,113 @@ func TestPropertyValueAggregate(t *testing.T) {
...
+		query: `{
+					"region":"us-west-2",
+					"propertyAlias":"/amazon/renton/1/rpm",
+					"aggregates":["SUM"],
+					"resolution":"1m"
+				}`,
+		expectedDescribeTimeSeriesWithContextArgs: &iotsitewise.DescribeTimeSeriesInput{Alias: Pointer("/amazon/renton/1/rpm")},
+		expectedMaxPages:   1,
+		expectedMaxResults: 0,
+		isDisassociated:    true,
```
[the remark this answers is not in the document]
Comment (idastambuk) [reply pkg/server/test/property_value_aggregate_test.go]: Good catch!

**Pull request merged**